outermysite/polls/imdbExactApi.py

```diff
@@ -0,0 +1,92 @@
+import requests
+import sys
+import traceback
+from bs4 import BeautifulSoup
+
+def getMovieResults(Id):
+    
+    url = "http://www.imdb.com/title/"+Id+"/?ref_=fn_al_tt_1"
+    
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        print "Unable to connect"
```

Comment:
No point in printing messages in a web server. Where are you going to look?

Log them instead. Try https://docs.python.org/2/howto/logging.html